src/pages/graphql/schema/store/queries/recaptcha-form-config.md

```diff
@@ -0,0 +1,94 @@
+---
+title: recaptchaFormConfig query
+edition: pwa
+---
+
+# recaptchaFormConfig query
+
+The `recaptchaFormConfig` query returns information about reCaptcha configuration for selected form type.
+
+## Syntax
+
+`recaptchaFormConfig(formType: ReCaptchaFormEnum!): ReCaptchaConfigOutput`
+
+## Example usage
+
+The following query returns information about the reCaptcha configuration for CONTACT form:
+
+**Request:**
+
+```graphql
+query {
+    recaptchaFormConfig(formType:CONTACT)
+		{
+    	is_enabled
+    	configurations {
+        re_captcha_type
+        badge_position
+        website_key
+        theme
+        language_code
+        minimum_score
+        validation_failure_message
+        technical_failure_message
+      }
+  }
+}
+```
+
+**Response:**
+
+If reCaptcha is enabled for particular form type, there are all the configuration details in the response:
+
+```json
+{
+  "data": {
+    "recaptchaFormConfig": {
+      "is_enabled": true,
+      "configurations": {
+        "re_captcha_type": "RECAPTCHA_V3",
+        "badge_position": "inline",
+        "website_key": "testtest",
+        "theme": "light",
+        "language_code": "en",
+        "minimum_score": 0.5,
+        "validation_failure_message": "Validation Failure Message",
+        "technical_failure_message": "Technical Failure Message"
+      }
+    }
+  }
+}
+```
+
+If reCaptcha is disabled for the form, the response provides the following information:
+
+```json
+{
+  "data": {
+    "recaptchaFormConfig": {
+      "is_enabled": false,
+      "configurations": null
+    }
+  }
+}
+```
+
+
```

Comment:
Delete the Input and Output attributes sections. Leave one blank line at the bottom of the file.